kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/FileNames.java

```diff
@@ -150,6 +161,10 @@ public static boolean isCommitFile(String fileName) {
         || UUID_DELTA_FILE_REGEX.matcher(filename).matches();
   }
 
+  public static boolean isChecksumFile(Path checksumFilePath) {
```

Comment:
It seems like all our other `isXXFile` methods take in a `String fileName` can we standardize on that? Unless there's a specific reason not to then feel free to object